src/commons/GitUtils.ts

```diff
@@ -31,13 +22,13 @@ export namespace GitUtils {
  * Execute git command script
  * @param params - command parameters
  */
-function executeGitScript(params: string): Set<string> {
+function executeGitScript(params: string): string {
   const currGitUrlScript = 'git config --get remote.origin.url';
   console.log(`Executing git command in: ${execSync(currGitUrlScript, { timeout: 30000 }).toString()}`);
   console.log(`Executing git command with ${params} param`);
-  const script: string = `git log ${params} --name-only --pretty="format:"`;
+  const script: string = `git ${params}`;
   console.log(`Executing: ${script}`);
   const res: string = execSync(script, { timeout: 30000 }).toString();
-  console.log(`Git command res: ${res ? res : 'No result returned!!!'}`);
-  return new Set<string>(res.split(/[\r\n]+/));
+  console.log(`Git command res: ${res ? res : 'No result!!!'}`);
+  return res ? res : '';
```

Comment:
don't you want to keep it undefined instead of empty string?

Reply:
done